pkg/config/config.go

```diff
@@ -56,6 +57,10 @@ type Marketplace struct {
 	InsecureClient bool   `env:"MARKETPLACE_HTTP_INSECURE_MODE" envDefault:"false"`
 }
 
+type ControllerReconcileSettings struct {
+	MeterDefControllerRequeueRate string `env:"METER_DEF_CONTROLLER_REQUEUE_RATE" envDefault:"3600"`
```

Comment:
```suggestion
	MeterDefControllerRequeueRate int `env:"METER_DEF_CONTROLLER_REQUEUE_RATE" envDefault:"3600"`
```
The library should convert it for you.

Reply:
@zach-source I had trouble using an int here, but I will try it again